requirements.txt

```diff
@@ -1,9 +1,10 @@
-torch==2.4.0
-torchaudio==2.4.0
+torch>=2.0.0
+torchaudio>=2.0.0
 tokenizers==0.21.0
-transformers==4.49.0
-huggingface_hub==0.28.1
-moshi==0.2.2
-torchtune==0.4.0
-torchao==0.9.0
-silentcipher @ git+https://github.com/SesameAILabs/silentcipher@master
\ No newline at end of file
+transformers
+huggingface_hub
+silentcipher @ git+https://github.com/SesameAILabs/silentcipher@master
+bitsandbytes>=0.41.1
+ffmpeg-python
+soundfile
+triton-cpu>=2.0.0
```

Comment:
Is there a reason moshi, torchtune, and torchao were removed? I expect this will break now, have you tested it on a GPU?